### PR TITLE
CNV-15055: Enhance Advanced storage settings section

### PR DESCRIPTION
**Pull request by bgaydosrh**
For 4.9, 4.10, 4.11, and 4.12.

Jira: https://issues.redhat.com/browse/CNV-15055
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2026545

Direct doc preview links:

https://49924--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-create-vms.html
 https://49924--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virt-edit-vms.html

Tagging @gouyang for Code/QE Review.

### CODE REVIEW

**Comment by gouyang** [comment]
LGTM

**Comment by bgaydosrh** [comment]
@gouyang - Ready for final recheck, thanks for your help.

Bob

File: modules/virt-storage-wizard-fields-web.adoc [hunk 1]
```diff
@@ -48,34 +48,42 @@
...
 |===
 
 [id="virt-storage-wizard-fields-advanced-web_{context}"]
 [discrete]
 == Advanced storage settings
 
-[cols="5a,3a,5a"]
-|===
-|Name | Parameter |  Description
+The following advanced storage settings are available for *Blank*, *Import via URL*, and *Clone existing PVC* disks. These parameters are optional. If you do not specify these parameters, the system uses the default values from the `kubevirt-storage-class-defaults` config map.
```
Comment (gouyang) [inline modules/virt-storage-wizard-fields-web.adoc:57]: the `kubevirt-storage-class-defaults` config map is removed from CNV 4.11 and 4.12.

Comment (gouyang) [reply modules/virt-storage-wizard-fields-web.adoc:57]: @bgaydosrh FYI

Comment (bgaydosrh) [reply modules/virt-storage-wizard-fields-web.adoc:57]: Thanks @gouyang - Is it safe to say that if the parms aren't specified that it uses the default values from the Storage Profile? Or would you prefer I just drop the last sentence referring to the map entirely? 
Bob

Comment (bgaydosrh) [reply modules/virt-storage-wizard-fields-web.adoc]: Thanks @gouyang - noted.

**Review by adellape — APPROVED** [review]

**Pull request merged**